src/main/java/org/sagebionetworks/web/client/widget/entity/controller/EntityActionControllerImpl.java

```diff
@@ -1642,8 +1642,6 @@ public void onComplete(TableUpdateTransactionResponse response) {
 						if (!errors.isEmpty()) {
 							view.showErrorMessage(errors);
 						} else {
-							fireEntityUpdatedEvent();
```

Comment:
We're using the PlaceChanger to go to the new version at line 1662, so this is unnecessary (and actually causes the page to break)